crawl-ref/source/job-data.h

```diff
@@ -77,6 +77,17 @@ static const map<job_type, job_def> job_data =
       { SK_THROWING, 2 }, { SK_WEAPON, 2 }, },
 } },
 
+{ JOB_BATTLEMAGE, {
+    "Ba", "Battlemage",
+    4, 4, 4,
+    { SP_TENGU, SP_KOBOLD, SP_BASE_DRACONIAN, SP_NAGA, SP_TROLL,
+      SP_MERFOLK, SP_DEMONSPAWN, },
+    { "leather armour plus:1", "book of War Wizardry" },
+    WCHOICE_NONE,
+    { { SK_FIGHTING, 1}, { SK_UNARMED_COMBAT, 3 }, { SK_DODGING, 2 },
+      { SK_SPELLCASTING, 2 }, { SK_TRANSMUTATIONS, 1 }, { SK_HEXES, 1 }, },
```

Comment:
Don't agree that physical combat backgrounds are weaker than book backgrounds in general. Fi and even Gl are among the stronger hellcrawl backgrounds imo, tho reasonable book combos are still good. It would be totally reasonable, to my mind, to remove assassin and allow hunter to start with darts. 

A funny (to me) idea I had earlier, re: hellcrawl magic: Replace transmutations magic with a non-magic transmutations skill and bring in a final fantasy legend-style meat system to do the transformations. Basically, monsters can drop a "chunk" that depends on monster type (like in the olden days) and doesn't rot (to avoid banking/scumming issues). Call them "monster hearts." With sufficient skill you can eat them to turn into the corresponding form permanently, but cancellable by eating other hearts. Funny part is that you can then claim to have food in the game. So much "flavor," prob better mechanically too.